### ISSUE TITLE
More than 2 GROUP BY not properly highlighted ?

### ISSUE THREAD

**Issue by tfrancart** [blake-regalia/linked-data.syntaxes#5]
![image](https://user-images.githubusercontent.com/2728945/58700711-237eda00-83a1-11e9-9c22-4cc0c6f51a7c.png)

**Comment by tfrancart** [comment]
Complete SPARQL query, if needed

```
# Qualité des rdfs:comment
# Multiple rdfs:comment sur une propriété
# rdfs:comment vides
# Héritage des propriétés, comment le présente-t-on ?
# Doit-on prendre les propriétés de Work ?
#
PREFIX owl:  	<http://www.w3.org/2002/07/owl#>
PREFIX rdfs: 	<http://www.w3.org/2000/01/rdf-schema#>
PREFIX skos:	<http://www.w3.org/2004/02/skos/core#>
PREFIX dcterms:	<http://purl.org/dc/terms/>
PREFIX cdm:		<http://publications.europa.eu/ontology/cdm#>

SELECT DISTINCT
	(REPLACE(STR(?class), "http://publications.europa.eu/ontology/cdm#", "cdm:") AS ?classShortUri)
	(REPLACE(STR(?property), "http://publications.europa.eu/ontology/cdm#", "cdm:") AS ?propertyShortUri)
	?propertyLabel
	(
	REPLACE(
		REPLACE(
			REPLACE(
				REPLACE(
					REPLACE(STR(?range), "http://publications.europa.eu/ontology/cdm#", "cdm:"),
					"http://www.w3.org/2001/XMLSchema#",
					"xsd:"
				),
				"http://www.w3.org/2000/01/rdf-schema#",
				"rdfs:"
			),
			"http://www.w3.org/1999/02/22-rdf-syntax-ns#",
			"rdf:"
		),
		"http://www.w3.org/2004/02/skos/core#",
		"skos:"
	)
	AS ?rangeShortUri
	)
	(GROUP_CONCAT(?comment; separator='. ') AS ?definition)
	?cardinality
	(STR(?deprecated) AS ?deprecie)
WHERE {
	# all classes in scope
    ?class a owl:Class .
    {
    	{ ?class rdfs:subClassOf* cdm:expression_publication_general . }
    	UNION
    	{ cdm:expression_publication_general rdfs:subClassOf* ?class . }
    }

	{
		{
			# all properties on the class
			?class rdfs:subClassOf ?r .
			?r a owl:Restriction .
			?r owl:onProperty ?property .

			# property label
			OPTIONAL { ?property rdfs:label ?propertyLabel . }
			OPTIONAL { ?property rdfs:comment ?comment . }

			# range + cardinality
			{
				{
					# someValuesFrom
					?r owl:someValuesFrom ?range .
					BIND("0..n" AS ?cardinality)
				}
				UNION
				{
					# qualifiedCardinality
					?r owl:qualifiedCardinality ?ca .
					{ {?r owl:onClass ?range .} UNION {?r owl:onDataRange ?range .} }
					BIND(CONCAT(STR(?ca), "..", STR(?ca)) AS ?cardinality)
				}
				UNION
				{
					# cardinality
					?r owl:cardinality ?ca .
					?property rdfs:range ?range .
					BIND(CONCAT(STR(?ca), "..", STR(?ca)) AS ?cardinality)
				}
				UNION
				{
					# maxQualifiedCardinality
					?r owl:maxQualifiedCardinality ?ca .
					{ {?r owl:onClass ?range .} UNION {?r owl:onDataRange ?range .} }
					BIND(CONCAT("0", "..", STR(?ca)) AS ?cardinality)
				}
				UNION
				{
					# maxCardinality
					?r owl:maxCardinality ?ca .
					?property rdfs:range ?range .
					BIND(CONCAT("0", "..", STR(?ca)) AS ?cardinality)
				}
				UNION
				{
					# minQualifiedCardinality
					?r owl:minQualifiedCardinality ?ca .
					{ {?r owl:onClass ?range .} UNION {?r owl:onDataRange ?range .} }
					BIND(CONCAT(STR(?ca), "..", "n") AS ?cardinality)
				}
				UNION
				{
					# minCardinality
					?r owl:minCardinality ?ca .
					?property rdfs:range ?range .
					BIND(CONCAT(STR(?ca), "..", "n") AS ?cardinality)
				}
			}
		}
		UNION
		{
			?property rdfs:domain ?class .
			?property rdfs:range ?range .

			# property label
			OPTIONAL { ?property rdfs:label ?propertyLabel . }
			OPTIONAL { ?property rdfs:comment ?comment . }

			BIND("0..n" AS ?cardinality)

			# Read on the property only if no explicit restriction exists on the class
			FILTER NOT EXISTS {
				?class rdfs:subClassOf ?r .
				?r a owl:Restriction .
				?r owl:onProperty ?property .
			}
		}
	}

	OPTIONAL { ?property owl:deprecated ?anything. BIND(true AS ?deprecated) }
}
GROUP BY ?class ?property ?propertyLabel ?range ?cardinality ?deprecated
ORDER BY ?class ?property
```

**Comment by blake-regalia** [comment]
Thanks again 🎉 Fixed in [v0.4.26](https://github.com/blake-regalia/linked-data.syntaxes/releases/tag/v0.4.26).
